integration_tests/pages/refer/findPerson.ts

```diff
@@ -1,11 +1,18 @@
 import Helpers from '../../support/helpers'
 import Page from '../page'
+import type { Prisoner } from '@prisoner-offender-search'
 
 export default class FindPersonPage extends Page {
   constructor() {
     super("Enter the person's identifier")
   }
 
+  searchForPrisonerNumber(prisonerNumber: Prisoner['prisonerNumber']) {
```

Comment:
Would it be better to use our internal language rather than the POS language: `searchForPerson(prisonNumber: Person['prisonNumber'])`?

Reply:
I went back and forth on this, as it's the Prisoner Offender Search we're searching in (this is the trouble with using two very similar but different terms!) but happy to update

Reply:
Well, we're testing that our service is able to search for something, no? Should the integration tests know how it does it? Are the integration tests not abstracted away from the 'how'?

Reply:
Have updated this to use the term `person`. There's a slight strangeness that we're passing in a `prisoner.prisonerNumber` when we define the type of the parameter as `Person['prisonNumber']` but under the hood they're the same thing so I don't think it's a problem. It also hides that implementation detail from users, as you say

Reply:
Could the person not be created further up in the test, then you can pass the person's prison number instead?

Reply:
I've merged this as-is as the "Person" doesn't exist at that point in our service and users will be reading the prison number from a record in DPS/NOMIS to enter it here and I found this to be a bit more of a logical flow.